spinners.json

```diff
@@ -870,5 +870,39 @@
 			"🌲", 
 			"🎄"
 		]
+	},
+	"blink": {
+		"interval": 75,
+		"frames": [
+			"(o)(o)", 
+			"(o)(o)", 
+			"(o)(o)", 
+			"(o)(o)", 
+			"(o)(o)", 
+			"(o)(o)", 
+			"(o)(o)", 
+			"(o)(o)", 
+			"(o)(o)", 
+			"(o)(o)", 
+			"(o)(o)", 
+			"(o)(o)", 
+			"(o)(o)", 
+			"(o)(o)", 
+			"(-)(-)", 
+			"(_)(_)"
```

Comment:
This doesn't really look like blinking eyes. And the blinking is too often. Any better blinking eyes ASCII animation out there?